src/Forms/Compare.cs

```diff
@@ -314,7 +319,11 @@ private void CompareSubtitles()
                     else
                     {
                         int columnsAlike = GetColumnsEqualExceptNumber(p1, p2);
-                        if (columnsAlike >= 0 && columnsAlike < 4)

```

Comment:
Oh yeah, I missed the coloring part, UPDATED.

the intention was to prevent calling **SetBackgroundColor** four times + all checkes if columns alike == 0